docs/design/kata-nydus-design.md

```diff
@@ -60,7 +60,7 @@ So in guest, container rootfs=overlay(`lowerdir=rafs`, `upperdir=snapshotdir/fs`
 
 > how to transfer the `rafs` info from `nydus-snapshotter` to the Kata Containers Containerd v2 shim?
 
-By default, when creating `OCI` image container, `nydus-snapshotter` will return [`struct` Mount slice](https://github.com/containerd/containerd/blob/main/mount/mount.go#L21) below to containerd and containerd use them to mount rootfs
+By default, when creating `OCI` image container, `nydus-snapshotter` will return [`struct` Mount slice](https://github.com/containerd/containerd/blob/main/core/mount/mount_linux.go#L36) below to containerd and containerd use them to mount rootfs
```

Comment:
The docs and the example below mentions `Mount` struct, not `mountOpts`. Maybe this is the right link?

https://github.com/containerd/containerd/blob/main/core/mount/mount.go#L30

Reply:
@beraldoleal changes applied